src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,182 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import {
+  FEMALE_SEX,
+  MALE_SEX,
+  ONE_CENTURY,
+} from '../utils/variables';
+import { PeopleSortType } from '../utils/PeopleSortType';
+
+export const getPreparedPeople = (
```

Comment:
I don't think you need export here unless you were planning to transfer the function to a separate file